tools/system_libs.py

```diff
@@ -246,6 +247,12 @@ def create_gl(libname): # libname is ignored, this is just one .o file
     check_call([shared.PYTHON, shared.EMCC, shared.path_from_root('system', 'lib', 'gl.c'), '-o', o])
     return o
 
+  # al
+  def create_al(libname): # libname is ignored, this is just one .o file
+    o = in_temp('al.o')
+    check_call([shared.PYTHON, shared.EMCC, shared.path_from_root('system', 'lib', 'al.c'), '-o', o])
```

Comment:
I wasn't sure what the best choice was, so I just made it exactly the same as the `create_gl` version. The functions contained inside are generally only called a few times at the beginning of a program and never again, so perhaps `-Os` would be best.